frontend/packages/ceph-storage-plugin/package.json

```diff
@@ -13,7 +13,8 @@
   "consolePlugin": {
     "entry": "src/plugin.ts",
     "integrationTestSuites": {
-      "ceph-storage": ["integration-tests/**/*.scenario.ts"]
+      "ceph-storage-install": ["integration-tests/**/installFlow.scenario.ts "],
```

Comment:
I should have done a PR about this but I had problem with installation because my setup was with 4 worker and didn't pass the installation.
We thought of this solution as it is less specific to names
```"consolePlugin": {
    "entry": "src/plugin.ts",
    "integrationTestSuites": {
      "ceph-storage": ["integration-tests/tests/installation/*.scenario.ts",
                                   "integration-tests/tests/*.scenario.ts",
                                  ]
    }
  }```
What do you think?

Reply:
There are some relative imports, so I didn't want to touch them so decided to do it with just test prefix instead of folder.